app/views/content_items/service_manual_guide.html.erb

```diff
@@ -78,5 +78,53 @@
       <%= render 'govuk_component/govspeak', content: @content_item.body.html_safe %>
     </div>
 
+    <div class="change-history" data-module="toggle">
+      <div class="change-history__latest-change">
+        <dl>
+        <% if @content_item.content_owners.any? %>
+          <dt>Published by:</dt>
+          <% @content_item.content_owners.each do |content_owner| %>
+            <dd>
+              <%= link_to content_owner.title, content_owner.href %>
+            </dd>
+          <% end %>
+        <% end %>
+        <% if @content_item.latest_change.present? %>
+          <dt>Last update:</dt>
+          <dd>
+            <% time = @content_item.latest_change.public_timestamp %>
+            <time datetime="<%= time.iso8601 %>" class="timestamp"><%= time.strftime("%e %B %Y") %></time>
+
+            <p class="change-history__summary">
+              <%= @content_item.latest_change.note %>
+            </p>
+
+            <div class="change-history__reason">
+              <%= simple_format(@content_item.latest_change.reason_for_change) %>
+            </div>
+          </dd>
+          <% end %>
+        </dl>
+      </div>
+    <% if @content_item.previous_changes.present? %>
+      <p><a href="#full-history" data-controls="full-history" data-expanded="false" role="button" aria-controls="full-history" aria-expanded="false">+ Show all page updates (<%= @content_item.previous_changes.length + 1 %>)</a></p>
+      <ol id="full-history" class="change-history__past js-hidden" aria-live="polite" role="region">
+      <% @content_item.previous_changes.each do |previous_change| %>
+        <li>
+          <% time = previous_change.public_timestamp %>
+          <time datetime="<%= time.iso8601 %>" class="timestamp"><%= time.strftime("%e %B %Y") %></time>
+          <p class="change-history__summary">
+            <%= previous_change.note %>
+          </p>
+
+          <div class="change-history__reason">
+            <%= simple_format(previous_change.reason_for_change) %>
+          </div>
```

Comment:
Maybe it would require a play to see if it aids readability rather than hinders it.. But this could go in a `_change_history.html.erb` partial.